src/libraries/reactotron/reactotronInstance.js

```diff
@@ -9,7 +9,7 @@ import {reactotronRecoilPlugin} from 'reactotron-recoil-plugin';
 export const reactotronInstance = reactotron
   .setAsyncStorageHandler(AsyncStorage) // AsyncStorage would either come from `react-native` or `@react-native-community/async-storage` depending on where you get it from
   .configure({
-    name: 'Better Social'
+    name: 'BetterSocialDev'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the name property from 'Better Social' to 'BetterSocialDev' seems to be a configuration change. Ensure that this new name is consistent with other parts of your application where this name might be used or displayed. If it's not, you may want to consider updating those as well.
```diff
-    name: 'Better Social'
+    name: 'BetterSocialDev'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->